sw/airborne/arch/stm32/subsystems/radio_control/spektrum_arch.c

```diff
@@ -47,7 +47,7 @@
 #define ONE_MHZ 1000000
 
 /* Number of low pulses sent to satellite receivers */
-#define MASTER_RECEIVER_PULSES 5
+#define MASTER_RECEIVER_PULSES 4     // RAB : Changed from 5 to 4 to enable the R100 satellite rx to bind successfully
```

Comment:
I know, I was just saying that this modification should be evaluated in an other pull request since it is not related to the main topic